### ISSUE TITLE
Feature Request: Support for APC RT 20000 RM XL - APC Management Module 

### ISSUE THREAD

**Issue by wriedel** [librenms/librenms#5504]
Hi Folks,

just wonder if it's possible to get a more detailed support for 3-Phase UPS like APC RT 20000 RM XL. 

I found a good description here:
https://fossies.org/linux/nut/drivers/apc-mib.c

I guess the relevant OIDs for 3-Phase UPS are:
snmpwalk -On -v1 -c COMMUNITY upsname PowerNet-MIB::upsPhaseOutputLoad.1.1
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.7.1.1.1 = INTEGER: 9320
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.7.1.1.2 = INTEGER: 6090
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.7.1.1.3 = INTEGER: 4720
snmpwalk -v1 -c COMMUNITY upsname PowerNet-MIB::upsPhaseOutputLoad.1.1
PowerNet-MIB::upsPhaseOutputLoad.1.1.1 = INTEGER: 9320
PowerNet-MIB::upsPhaseOutputLoad.1.1.2 = INTEGER: 6090
PowerNet-MIB::upsPhaseOutputLoad.1.1.3 = INTEGER: 4720

There are also MaxLoad and MinLoad values returned, which might be useful for defining alert limits:
Max
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.8.1.1.1 = INTEGER: 12990
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.8.1.1.2 = INTEGER: 7200
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.8.1.1.3 = INTEGER: 7880

Min
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.9.1.1.1 = INTEGER: 8690
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.9.1.1.2 = INTEGER: 5490
.1.3.6.1.4.1.318.1.1.1.9.3.3.1.9.1.1.3 = INTEGER: 4240

Thank you,
Wolfgang

DO NOT DELETE THIS INFORMATION.

> Please read this information carefully.

GitHub issues is for feature requests or bugs, please do not post issues asking for help or how to do X, Y or Z. 
You can use our irc channel ##librenms on freenode to ask questions or our [community site](https://community.librenms.org).

Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [x] Is your install up to date? [Updating your install](http://docs.librenms.org/General/Updating/)
      Please do not submit an issue if your install is not up to date within the last 24 hours or on a stable monthly release.
- [ x Please include all of the information between the `====================================` section of `./validate.php` which you can run from the cli.
- [x] Unless your issue is for a WebUI fix or feature then please provide ALL info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [x] Please provide as much detail as possible.
- [x] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

Failed to parse MIB file /usr/share/snmp/mibs
==========================================================
Component | Version
--------- | -------
LibreNMS  | 6df39b9bbdd5e6a0184640f3c328253f09e8bcab
DB Schema | 156
PHP       | 7.0.14-2
MySQL     | 10.2.3-MariaDB-10.2.3+maria~sid-log
RRDTool   | 1.6.0
SNMP      | NET-SNMP 5.7.3
==========================================================

[OK]    Database connection successful

**Comment by wriedel** [comment]
attached some more debugs:
[apc-PowerNetMIB-surt2k-snmpwalk.txt](https://github.com/librenms/librenms/files/716666/apc-PowerNetMIB-surt2k-snmpwalk.txt)
[apc-surt2k-snmpwalk.txt](https://github.com/librenms/librenms/files/716664/apc-surt2k-snmpwalk.txt)
[apc-surt2k-discovery.txt](https://github.com/librenms/librenms/files/716667/apc-surt2k-discovery.txt)
[apc-surt2k-poller.txt](https://github.com/librenms/librenms/files/716665/apc-surt2k-poller.txt)

**Comment by laf** [comment]
Can you please test: #5504

**Comment by wriedel** [comment]
Hi Neil,
many thanks for picking this up. Currently it's not really working for me. I have two apc-surt-20000 and one is now showing voltage for two phases while the other still for one while in reality I have three phases
here some debugs:
http://pastebin.com/0KDGyugw
http://pastebin.com/eYuwv5Vi
and SNMP walks
http://pastebin.com/PuT83sTP
http://pastebin.com/5tVnjQuR
Thank you,
Wolfgang

**Comment by laf** [comment]
Can you please put the comment in: https://github.com/librenms/librenms/pull/5558

**Comment by laf** [comment]
For now classing as duplicate of #2733

Once #5558 is merged you can see if this provides what you want.
